admin/zabbix/Makefile

```diff
@@ -12,7 +12,8 @@ PKG_VERSION:=6.2.3
 PKG_RELEASE:=3
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
-PKG_SOURCE_URL:=https://cdn.zabbix.com/zabbix/sources/stable/6.2/
+PKG_SOURCE_URL:=https://cdn.zabbix.com/zabbix/sources/stable/6.2/ \
```

Comment:
Why keep the broken url ?

Reply:
If the package is updated to 6.4 (the current stable) then the stable URL would apply.

Reply:
```suggestion
PKG_SOURCE_URL:=https://cdn.zabbix.com/zabbix/sources/stable/$(basename $(PKG_VERSION))/ \
```

This looks better.

Reply:
> This might be better.

I have made this change for both URLs, though I prefer the way it is done in [kernel-version.mk](https://github.com/openwrt/openwrt/blob/abcb30d36cfe65e3ed7786c929c1a2350dd2a9c2/include/kernel-version.mk#L46).